recipes/e3sm_to_cmip/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "e3sm_to_cmip" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  #sha256: a23d7440bab9950f6fef8a6e5fe60741e4e5be2a2421d37ca9a1739dab1deef1
+  git_url: git://github.com/E3SM-Project/e3sm_to_cmip.git
+  git_tag: master
+
+build:
+  skip: True  # [win or py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - nco
+    - cmor >=3.5.0
+    - cdutil
+    - cdms2 >=3.1
+    - progressbar2
+    - pathos
+    - pyyaml
+    - xarray
+    - netcdf4
+    - dask
+    - scipy
+
+test:
+  commands:
+    - e3sm_to_cmip --help
+  imports:
+    - e3sm_to_cmip
+
+about:
+  home: https://github.com/E3SM-Project/e3sm_to_cmip
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A cli utility to transform E3SM model output into CMIP compatible data.'
+  description: |
+    Transform e3sm time series variables into cmip compatible data. Each
+    variable needs its own handler script, implemented in the cmor_handlers
+    directory (see directory for current handlers). In addition, you will need
+    to clone the cmor repo (https://github.com/PCMDI/cmor) to access the Test
+    and Tables directories. Test holds the common_user_input.json file which
+    can be used as a placeholder for the user supplied metadata, and Tables
+    holds all the CMIP6 variable tables.
+  doc_url: https://github.com/E3SM-Project/e3sm_to_cmip
+  dev_url: https://github.com/E3SM-Project/e3sm_to_cmip
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - sterlingbaldwin
```

Comment:
@sterlingbaldwin, please acknowledge that you're willing to be a maintainer.

Reply:
I so acknowledge.